Tweak/Helper/KayokoHelper.xm

```diff
@@ -132,27 +140,133 @@ static void override_UIKeyboardLayoutStar_didMoveToWindow(UIKeyboardLayoutStar*
     CFNotificationCenterPostNotification(CFNotificationCenterGetDarwinNotifyCenter(), (CFStringRef)kNotificationKeyCoreHide, nil, nil, YES);
 }
 
+%group DictationAppearance
```

Comment:
Use substrate and not logos.

Reply:
I just don't want to pain to write long method signatures. Sorry.